app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -135,6 +135,17 @@ class EventsFragment : Fragment(), ScrollToTop {
                 showNoInternetScreen(!isConnected && eventsViewModel.events.value == null)
             })
 
+        eventsViewModel.appLinkOrderId
+            .nonNull()
+            .observe(viewLifecycleOwner, Observer {
```

Comment:
@iamareebjamal @anhanh11001 I need help here. It is not observing the data for which I set value from the main activity. How to pass live data from the activity to events fragment? 
Last opting I got is shared preference, but I don't think it's good to use that.